### PR TITLE
Add presenter for search results dictionary

### PR DESCRIPTION
**Pull request by tombye**
To be used like so:

```
from helpers.presenters import SearchResults

search_results = presenters.SearchResults(res)
total = search_results.get_total()
services = search_results.get_services()
```

Both methods return dictionaries.

### CODE REVIEW

File: app/helpers/presenters.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        #results_json = open(os.path.join(os.path.dirname(__file__), 'results.json'), 'r').read()
+        #results_dict = json.loads(results_json)
+
+        self.lots = {
+            'IaaS': 'Infrastructure as a Service',
+            'SaaS': 'Software as a Service',
+            'PaaS': 'Platform as a Service',
+            'SCS': 'Specialist Cloud Services'
+        }
+        self.total = results_dict['hits']['total']
```
Comment (TheDoubleK) [inline app/helpers/presenters.py]: This line gives me an error `TypeError: string indices must be integers, not str`

**Comment by TheDoubleK** [comment]
Also the import that I got to work (in views) was:
`from app.helpers import presenters`
It doesn't like me trying to `import presenters.SearchResults`

**Comment by tombye** [comment]
I've fixed the issue and plugged it into the /search view so it should both work and produce some output to prove it.

**Comment by TheDoubleK** [comment]
Looks good - or at least better than it was!

**Pull request merged**